contracts/src/interfaces/ITeleporterTokenDestination.sol

```diff
@@ -27,6 +27,8 @@ struct TeleporterTokenDestinationSettings {
     address teleporterManager;
     bytes32 sourceBlockchainID;
     address tokenSourceAddress;
+    uint8 tokenSourceDecimals;
```

Comment:
Add natspec descriptions of these new fields

Reply:
addressed in https://github.com/ava-labs/teleporter-token-bridge/pull/142/commits/c45fbc1f7b6152b9532240d729e62641dad3ab8a